Formula/wownero.rb

```diff
@@ -5,6 +5,7 @@ class Wownero < Formula
       tag:      "v0.9.3.3",
       revision: "e2d2b9a447502e22467af9df20e0732b3dd4ac4c"
   license "BSD-3-Clause"
+  revision 1
```

Comment:
This reports no linkage with `protobuf`, so it's unlikely that a revision bump is needed.